modules/custom/movie_reservation/movie_reservation.routing.yml

```diff
@@ -4,4 +4,6 @@ movie_reservation.movieReservation:
     _controller: '\Drupal\movie_reservation\Controller\MovieReservationController::movieReservation'
     _title: 'Movie Reservation'
   requirements:
-    _permission: 'access content'
\ No newline at end of file
+    _permission: 'access content'
+  methods: [GET]
+  
```

Comment:
Ako smo recimo ostavili komentar na jednom mestu za prazne redove, to ne znaci da se ispravi samo tu gde je komentar, vec i na ostalim mestima gde se javlja ista stvar.